Contest_API.md

```diff
@@ -538,6 +538,38 @@ meaning that all conditions must be met (they are logically `AND`ed).
 Note that filtering on any other field, including fields with the type
 array of ID, does not have to be supported.
 
+### API version
```

Comment:
If we add more fields in the future the name `version` might not be ideal. But we can change it at that point, that doesn't impact backwards compatibility.